packages/svelte/tests/runtime-runes/samples/invalid-html-ssr/_config.js

```diff
@@ -0,0 +1,36 @@
+import { test } from '../../test';
+
+let console_error = console.error;
+
+/**
+ * @type {any[]}
+ */
+const log = [];
+
+export default test({
+	compileOptions: {
+		dev: true
+	},
+
+	html: `<p></p><h1>foo</h1><p></p>`,
+
+	recover: true,
```

Comment:
Why is it important for the test to recover? Shouldn't the console.error happen before it is thrown?

Reply:
It does, but the test will than fail as we run into an exception when hydrating causing the test to fail.

Reply:
oh right